sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerContext.java

```diff
@@ -341,6 +346,14 @@ public String getSchemaResourceType(String schema, String resourceName)
     return plannerToolbox.rootSchema().getResourceType(schema, resourceName);
   }
 
+  /**
+   * Returns the set of lookups to load for a given task.
+   */
+  public Set<String> getLookupsToLoad()
```

Comment:
Shouldn't this also use `LookupLoadingSpec`?

Reply:
Have made the change.